File: CPP/Orang.cpp
#pragma once
#include <iostream>
#include <string>


using namespace std;


class Orang
{
private:
    string nama;
    int usia;
    string alamat;
    string noTelp;
public:
    Orang()
    {


    }


    Orang(string nama, int usia, string alamat, string noTelp)
    {
        this->nama = nama;
        this->usia = usia;
        this->alamat = alamat;
        this->noTelp = noTelp;
    }

    // Getter methods
    string getNama() {
        return this->nama;
    }

    int getUsia() {
        return this->usia;
    }

    string getAlamat() {
        return this->alamat;
    }

    string getNoTelp() {
        return this->noTelp;
    }

    // Setter methods
    void setNama(string nama) {
        this->nama = nama;
    }

    void setUsia(int usia) {
        this->usia = usia;
    }

    void setAlamat(string alamat) {
        this->alamat = alamat;
    }

    void setNoTelp(string noTelp) {
        this->noTelp = noTelp;
    }

    ~Orang()
    {


    }
};

File: PY/Orang.py
class Orang:
    __nama = ""
    __usia = 0
    __alamat = ""
    __noTelp = ""
    
    def __init__(self, nama="", usia=0, alamat="", noTelp=""):
        self.__nama = nama
        self.__usia = usia
        self.__alamat = alamat
        self.__noTelp = noTelp
    
    # Getter methods
    def getNama(self):
        return self.__nama
    
    def getUsia(self):
        return self.__usia
    
    def getAlamat(self):
        return self.__alamat
    
    def getNoTelp(self):
        return self.__noTelp
    
    # Setter methods
    def setNama(self, nama):
        self.__nama = nama
    
    def setUsia(self, usia):
        self.__usia = usia
    
    def setAlamat(self, alamat):
        self.__alamat = alamat
    
    def setNoTelp(self, noTelp):
        self.__noTelp = noTelp

